troposphere/codebuild.py

```diff
@@ -143,6 +143,8 @@ def validate(self):
         valid_types = [
             'LINUX_CONTAINER',
             'WINDOWS_CONTAINER',
+            'ARM_CONTAINER',
```

Comment:
Minor change...can you alphabetize this list? Thanks!

Reply:
Done ✅